Error Updating Under Win 10 x64

(hopefully not a duplicate #118 and #119, since this is make-related error)

    ruby 2.5.3p105 (2018-10-18 revision 65156) [x64-mingw32]

When I try to update the gem under Win 10 x64 I get this error:

```
current directory: C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.1/ext/bigdecimal/util
C:/tools/ruby25/bin/ruby.exe -r ./siteconf20181225-9440-zwbdav.rb extconf.rb
checking for Windows... yes
creating Makefile

current directory: C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.1/ext/bigdecimal/util
make "DESTDIR=" clean
bash.exe: warning: could not find /tmp, please create!

current directory: C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.1/ext/bigdecimal/util
make "DESTDIR="
generating util-x64-mingw32.def
bash.exe: warning: could not find /tmp, please create!
compiling util.c
linking shared-object bigdecimal/util.so
x86_64-w64-mingw32-gcc.exe: error: C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.1/tmp/x64-mingw32/bigdecimal/2.5.3/bigdecimal.so: No such file or directory
make: *** [Makefile:259: util.so] Error 1

make failed, exit code 2
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -7,6 +7,7 @@ build_script:
   - rake -rdevkit compile
 test_script:
   - rake test
+  - rake install
 deploy: off
 environment:
   matrix:

--- a/ext/bigdecimal/util/extconf.rb
+++ b/ext/bigdecimal/util/extconf.rb
@@ -4,11 +4,11 @@ require 'mkmf'
 checking_for(checking_message("Windows")) do
   case RUBY_PLATFORM
   when /cygwin|mingw/
-    if defined?($extlist)
-      build_dir = "$(TARGET_SO_DIR)../"
-    else
+    if ARGV.include?('-rdevkit') # check `rake -rdevkit compile` case
       base_dir = File.expand_path('../../../..', __FILE__)
       build_dir = File.join(base_dir, "tmp", RUBY_PLATFORM, "bigdecimal", RUBY_VERSION, "")
+    else
+      build_dir = "$(TARGET_SO_DIR)../"
     end
     $libs << " #{build_dir}bigdecimal.so"
     true

